### PR TITLE
Add plnsvc-codecov secret

### PR DESCRIPTION
**Pull request by Roming22**
This secret is used to report code coverage on pipeline service repositories

### CODE REVIEW

File: components/tekton-ci/production/kustomization.yaml [hunk 1]
```diff
@@ -4,6 +4,7 @@ resources:
 - ../base
 - ../base/external-secrets
 - plnsvc-ci-secret.yaml
+- plnsvc-codecov-secret.yaml
```
Comment (xinredhat) [inline components/tekton-ci/production/kustomization.yaml:7]: @Roming22 it should be `plnsvc-codecov.yaml`

Comment (enarha) [reply components/tekton-ci/production/kustomization.yaml:7]: You better rename plnsvc-codecov.yaml to plnsvc-codecov-secret.yaml

Comment (Roming22) [reply components/tekton-ci/production/kustomization.yaml:7]: Fixed

**Review by xinredhat — APPROVED** [review]

**Review by xinredhat — COMMENTED** [review]
/lgtm

**Pull request merged**